modules/quantumBidAdapter.js

```diff
@@ -297,23 +297,22 @@ export const spec = {
    * Register the user sync pixels which should be dropped after the auction.
    *
    * @param {SyncOptions} syncOptions Which user syncs are allowed?
-   * @param {ServerResponse[]} serverResponses List of server's responses.
+   * @param {ServerResponse} serverResponse A successful response from the server
    * @return {UserSync[]} The user syncs which should be dropped.
    */
-  getUserSyncs: function (syncOptions, serverResponses) {
-    const syncs = []
-    if (syncOptions.iframeEnabled) {
-      syncs.push({
-        type: 'iframe',
-        url: '//acdn.adnxs.com/ib/static/usersync/v3/async_usersync.html'
-      });
-    }
-    if (syncOptions.pixelEnabled && serverResponses.length > 0) {
-      syncs.push({
-        type: 'image',
-        url: serverResponses[0].body.sync[0]
-      });
-    }
+  getUserSyncs: function (syncOptions, serverResponse) {
+    const syncs = [];
+    utils._each(serverResponse, function(serverResponse) {
+      if (serverResponse.body && serverResponse.body.sync) {
+        const syncType = syncOptions.pixelEnabled ? 'image' : 'iframe';
```

Comment:
To clarify the intent here - do you want to only use one type of pixel per auction?

Further, the `userSync` feature defaults the `pixelEnabled` field to `true` and the iframe pixels are opt-in via the config.  So your iframe pixels will only drop with specific config options; ie 
```
userSync: {
  pixelEnabled: false,
  iframeEnabled: true
}
```
My assumption is this would create a heavier bias towards image-based pixels.  Is that a concern?

Reply:
Hi. Thank you for your review. From our side all sync pixels are images. I will modify to send all the time 'pixel' type.